src/gui/gui.h

```diff
@@ -304,24 +335,44 @@ class GUI final {
     bool m_showInterruptsScaler = false;
     Widgets::Log m_log = {settings.get<ShowLog>().value};
     struct MemoryEditorWrapper {
-        MemoryEditorWrapper(GUI *gui) {
+        MemoryEditorWrapper(GUI *gui, bool &show, size_t &offsetAddr, size_t baseAddr = 0x0000)
+            : m_show(show), m_offsetAddr(offsetAddr), m_baseAddr(baseAddr) {
             editor.OptShowDataPreview = true;
             editor.OptUpperCaseHex = false;
             editor.PushMonoFont = [gui]() { gui->useMonoFont(); };
         }
-        MemoryEditor editor;
+        bool &m_show;
+        size_t &m_offsetAddr;
+        size_t m_baseAddr;
```

Comment:
Probably should be `const`.

Reply:
Actually you know what, since this needs to be in the wrapper too it might as well also be a reference down to the hex editor class, just a constant one. Is that okay?

Reply:
Well, the advantage of a reference is the ability to change it from somewhere else, but it'll otherwise still cost the dereferencing impact. If the value truly will never change, then we wouldn't gain anything from it being a reference, and will only pay its price.

Reply:
Roger that. I can make the wrapper member const as well as inside the hex editor.